src/UploadTaskQueuer.cpp

```diff
@@ -1,43 +0,0 @@
-/*
- * Copyright (C) 2017 Prism Skylabs
- */
-#include "private/UploadTaskQueuer.h"
-
-namespace prism
-{
-namespace connect
-{
-
-Status SimpleUploadTaskQueuer::addBackgroundTask(UploadBackgroundTaskPtr task)
```

Comment:
For now queuer simpy pushes tasks into queue, so I've inlined this method reducing it to enqueueTask() method of ArtifactUploader::Impl(). This eliminated two files, one class and three methods per single client method.